src/main/java/xyz/nucleoid/plasmid/game/ManagedGameSpace.java

```diff
@@ -270,7 +271,9 @@ private void onRemovePlayer(ServerPlayerEntity player) {
     public CompletableFuture<StartResult> requestStart() {
         return Scheduler.INSTANCE.submit(server -> {
             try {
-                return this.invoker(RequestStartListener.EVENT).requestStart();
+                StartResult startResult = this.invoker(RequestStartListener.EVENT).requestStart();
+                GameEvents.START_REQUEST.invoker().onRequestStart(this, startResult);
```

Comment:
Should these events have the ability to replace the start result? It might be worth also noting in the documentation that they can be called with a failed start result.

Reply:
Being able to override the start result could be useful